daprdocs/content/en/reference/api/jobs_api.md

```diff
@@ -0,0 +1,147 @@
+---
+type: docs
+title: "Jobs API reference"
+linkTitle: "Jobs API"
+description: "Detailed documentation on the jobs API"
+weight: 1300
+---
+
+{{% alert title="Note" color="primary" %}}
+Jobs is currently in alpha.
+{{% /alert %}}
+
+With the jobs API, you can schedule jobs and tasks in the future.
+
+## Schedule a job
+
+Schedule a job with a name.
+
+```
+POST http://localhost:3500/v1.0-alpha1/jobs/<name>
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`name` | Name of the job you're scheduling
+`data` | A string value and can be any related content. Content is returned when the reminder expires. For example, this may be useful for returning a URL or anything related to the content.
+`dueTime` | Specifies the time after which the jobs are invoked. Its format should be [time.ParseDuration](https://pkg.go.dev/time#ParseDuration)
+
+### Request body
+
+```json
+{
+  "job": {
+      "data": {
+          "@type": "type.googleapis.com/google.type.Expr",
+          "expression": "<expression>"
+      },
+      "dueTime": "30s"
+  }
+}
+```
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or Scheduler control plane service
+
+### Response content
+
+The following example curl command creates a job, naming the job `test1` and specifying the `dueTime` and the `data`.
+
+```bash
+$ curl -X POST \
+  http://localhost:3500/v1.0-alpha1/jobs/test1 \
```

Comment:
```suggestion
  http://localhost:3500/v1.0-alpha1/jobs/jobforjabba \
```

Reply:
I thought we should have a bit of fun, given that In "Star Wars: A New Hope," Han Solo mentions a job he had with Jabba the Hutt!